src/twisted/internet/test/test_tls.py

```diff
@@ -390,16 +389,6 @@ class AbortSSLConnectionTests(
     requiredInterfaces = (IReactorSSL,)
     endpoints = SSLCreator()
 
-    def buildReactor(self):
```

Comment:
No longer necessary since we're now making the global reactor match the temporary one.